Makefile

```diff
@@ -0,0 +1,52 @@
+#! /usr/bin/make -f
+
+# Project variables.
+PROJECT_NAME = 'ignite apps'
+
+## govet: Run go vet.
+govet:
+	@echo Running go vet...
+	@go list -f '{{.Dir}}/...' -m | xargs go vet 
+
+## govulncheck: Run govulncheck
+govulncheck:
+	@echo Running govulncheck...
+	@go list -f '{{.Dir}}/...' -m | grep -v integration |xargs go run golang.org/x/vuln/cmd/govulncheck
+
+## format: Install and run goimports and gofumpt
+format:
+	@echo Formatting...
+	@go run mvdan.cc/gofumpt -w .
+	@go run golang.org/x/tools/cmd/goimports -w -local github.com/ignite/cli .
+	@go run github.com/tbruyelle/mdgofmt/cmd/mdgofmt -w docs
+
+## lint: Run Golang CI Lint.
+lint:
+	@echo Running gocilint...
```

Comment:
nit: typo

Reply:
I couldn't find any typo in the section you highlighted. Could you make it clearer?

Reply:
I could not find typo error. could you give me more information ?

Reply:
The tool is called golangci-lint